build-logic/settings.gradle.kts

```diff
@@ -0,0 +1,20 @@
+@file:Suppress("UnstableApiUsage")
+
+pluginManagement {
+    repositories {
+        gradlePluginPortal()
+        google()
+        mavenCentral()
+    }
+}
+
+dependencyResolutionManagement {
+    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
+    repositories {
+        google()
+        mavenCentral()
+    }
+    versionCatalogs.create("libs").from(files("../gradle/libs.versions.toml"))
```

Comment:
I wonder, do this line create catalog for all projects?

Reply:
no, only for build-logic (and any subproject within build-logic project)
for the subprojects within main (Elmslie) project it is created implicitly by Gradle